.github/workflows/ci-data.yml

```diff
@@ -0,0 +1,53 @@
+name: GCS Insert
+
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  gcs_insert:
+    runs-on: ubuntu-latest
+    steps:
+      - name: set date
+        id: set_date
+        run: |
+          set -euo pipefail
+
+          echo "::set-output name=yesterday::$(date --date=yesterday +%Y-%m-%d)"
+
+      - name: generate data
+        id: generate_data
+        run: |
+          set -euo pipefail
+
+          yesterday="${{ steps.set_date.outputs.yesterday }}"
+
+          [ -n "$yesterday" ] || exit 1
+
+          gh api --paginate "repos/ibis-project/ibis/actions/runs?per_page=100&created=${yesterday}" --jq '.workflow_runs[]' > workflows.json
```

Comment:
Does it use the `GITHUB_TOKEN` provided by github actions? In that case the rate limit is just 1000, so sleeping one second won't help much.

Reply:
Fixed.